hack/testing/check-EFK-running.sh

```diff
@@ -130,60 +130,83 @@ else
 fi
 
 TEST_DIVIDER="-------------------------------------------------------"
+
+function echo_divider() {
+  msg=${1:-""}
+  echo $TEST_DIVIDER
+  echo "$TEST_DIVIDER $msg"
```

Comment:
use
```
   echo "$TEST_DIVIDER ${1:-}"
```
That will print $1 if it is set, otherwise, it will print an empty string